NASA-AMMOS/aerie-ui
# Add derived start to directives table and derived start/end to spans table

This resolves #579 

To test:

1. Open a plan with directives and spans
2. Open the `Activity Directives Table` panel
3. Click on the `...` button to expand the list of viewable columns
![Screenshot 2023-04-28 at 3 16 40 PM](https://user-images.githubusercontent.com/5290214/235263154-78cd2eb9-9e6d-4ce2-8795-ffc90fe0beee.png)
4. Select the `Derived Start Time` column
5. Verify that the new column is displayed with the correct values (you may need to adjust the column sizes/scroll to see the new column)
6. Open the `Simulated Activities Table` panel
7. Repeat steps 2 - 4, but also include the `Derived End Time` column
8. Verify that the start/end time columns are displayed with the correct values.
